Assets/MixedRealityToolkit/_Core/EventDatum/Input/InputEventData.cs

```diff
@@ -16,7 +17,14 @@ public class InputEventData : BaseInputEventData
         /// <summary>
         /// Handedness of the <see cref="IMixedRealityInputSource"/>.
         /// </summary>
-        public Handedness Handedness { get; private set; } = Handedness.None;
+        public Handedness Handedness
+        {
+            get
+            {
+                var controller = MixedRealityToolkit.InputSystem.TryGetController(InputSource);
```

Comment:
I suppose with this change we should also remove the handedness property in `Initialize`

Reply:
Or we can remove `eventData.Handedness` all together

Reply:
Strongly against removing `eventData.Handedness`. It provides easy, discoverable access to data that is valuable when performing handed interactions.

Reply:
Pointers shouldn't know about hands so it wouldn't matter.